### PR TITLE
Books with modules

### PR DESCRIPTION
**Pull request by dazehinn**
Refactored the Awesome books using:
* ES6 modules
* arrow functions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      <div class="main-links">
+        <ul>
+          <li><a id="listshow" href="#">List</a></li>
+          <li><a id="addshow" href="#">Add New</a></li>
+          <li><a id="contactshow" href="#">Contact</a> </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <div id="date-time">fecha</div>
```
Comment (ridwanediallo) [inline index.html]: - [ ] Great job team 👏. However, I believe you forgot to include the current date; please work on that as it is one of the requirements to have it displayed below the navigation bar. 👌So please use the [luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library for this.

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi team, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️

- [ ] Please, in your readme file, in the setup section, show the user how to get your project set up on their local environment.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
```
Comment (rloterh) [inline index.html]: - [ ] Displayed date is not properly formatted, and hence not displaying as intended, kindly ensure to fix it:

**Incorrect date format:**
image

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: needs attention!

File: modules/awesomebooks.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  static addBook(e) {
+    if (title.value.length > 0 && author.value.length > 0) {
+      e.preventDefault();
+      booksArray.push(new Book(title.value.toUpperCase(), author.value.toUpperCase()));
+      title.value = '';
+      author.value = '';
+      AwesomeBooks.showList();
+      localStorage.setItem('books', JSON.stringify(booksArray));
+    }
+  }
```
Comment (rloterh) [inline modules/awesomebooks.js:30]: -  [ ] OPTIONAL
  * Books are stored correctly in the local storage when a book detail is entered, however when the page is refreshed the book remains in the local storage but however vanishes from the list of books page. Please ensure that data is both correctly displayed and reflected in the local storage even when the page is reloaded.

  * Also, it's not required to change data to uppercase before adding it to the local storage

**Missing book list:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Nshanji, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
- Good job with refactoring so far!
- Good ReadMe

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @dazehinn,

✔️ Date is well implemented and displayed

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**